docs/usage/oauth.md

```diff
@@ -104,7 +104,7 @@ app.get('/auth/callback', async (req, res) => {
 ```
 </details>
 
-After process is completed, you can navigate to `{your ngrok address}/oauth/begin` in your browser to begin OAuth. When it completes, you will have a Shopify session that enables you to make requests to the Admin API, as detailed next.
+After process is completed, you can navigate to `{your ngrok address}/auth/callback` in your browser to begin OAuth. When it completes, you will have a Shopify session that enables you to make requests to the Admin API, as detailed next.
```

Comment:
I think we should actually be going to `/login` here, though?

Reply:
+1, the actual endpoint in the app should be `/login`, which will initiate the auth flow.